### PR TITLE
Prep for 4.14 and 5.0.5

### PR DESCRIPTION
**Pull request by yadij**

### CODE REVIEW

**Comment by yadij** [comment]
@rousskov, @kinkie could one of you guys please review this ASAP for anubis fast-track. If not I will have to bump the releases out to Feb 9.

File: ChangeLog [hunk 1]
```diff
@@ -1,3 +1,13 @@
+Changes in squid-5.0.5 (02 Feb 2021):
+
+	- HTTP: Do not send keep-alive or close in HTTP Upgrade requests
```
Comment (rousskov) [inline ChangeLog:3]: Just a clarifying suggestion that may also improve folks ability to find relevant changes. Feel free to ignore.

```suggestion
	- HTTP: Do not send Connection:keep-alive/close in HTTP Upgrade requests
```

**Review by rousskov — APPROVED** [review]
I am approving this simple change without a close review to put it on the fast track, per @yadij  request. @kinkie, please feel free to request changes, of course!

However, I would like to use this opportunity to say that, IMO, copyright year changes in v4 (8e56335) and v5 (04f296e) should not have been committed until the resolution of https://github.com/squid-cache/squid/pull/763#pullrequestreview-579383507 discussion. It is very unpleasant for me to see changes shoveled into the official code like that.

**Comment by yadij** [comment]
> I am approving this simple change without a close review to put it on the fast track, per @yadij request. @kinkie, please feel free to request changes, of course!
> 
> However, I would like to use this opportunity to say that, IMO, copyright year changes in v4 and v5 should not have been committed until the resolution of [#763 (review)](https://github.com/squid-cache/squid/pull/763#pullrequestreview-579383507) discussion. It is very unpleasant for me to see changes shoveled into the official code like that.

FTR: The above statement implies that the copyright renewal is an unusual occurrence. Where in fact it is a regular event since the copyright restructuring project in 2014. It is performed annually by squid-core approved automation.

**Comment by yadij** [comment]
Marking for Anubis when the requested ChangeLog modification plus re-do of build checks.

**Comment by rousskov** [comment]
> Alex: IMO, copyright year changes in v4 and v5 should not have been committed until the resolution of [#763 (review)](https://github.com/squid-cache/squid/pull/763#pullrequestreview-579383507) discussion. It is very unpleasant for me to see changes shoveled into the official code like that.

> Amos: The above statement implies that the copyright renewal is an unusual occurrence

There is no such implication.

> it is a regular event since the copyright restructuring project in 2014. It is performed annually by squid-core approved automation.

If you think that this technicality can somehow justify what happened, then, technically, Squid vX modifications that are also relevant to the master branch must happen in master first. The existence of a modification script is not relevant to that key principle, of course.

**Comment by yadij** [comment]
> > Alex: IMO, copyright year changes in v4 and v5 should not have been committed until the resolution of [#763 (review)](https://github.com/squid-cache/squid/pull/763#pullrequestreview-579383507) discussion. It is very unpleasant for me to see changes shoveled into the official code like that.
> 
> > Amos: The above statement implies that the copyright renewal is an unusual occurrence
> 
> There is no such implication.
> 

Then please clarify exactly what you meant by that last sentence. It is filled with emotive negative wording.

This whole discussion gives the impression that something surprising has happened that you are only now

and the topic of the paragraph is about the *routine maintenance* commits on v4 and v5 branches.


> > it is a regular event since the copyright restructuring project in 2014. It is performed annually by squid-core approved automation.
> 
> If you think that this technicality can somehow justify what happened, then, technically, Squid vX modifications that are also relevant to the master branch must happen in master first. The existence of a modification script is not relevant to that key principle, of course.

The automatic maintenance on each branch is independent. The scripts/source-maintenance.sh in each vN branch are what is run and the resulting changes merged to that vN branch after the branch maintainer (a.k.a. me) reviews.

You should well know how things work with Squid code management. I'm not sure why I have to explain this, except that you called seeing the scripts in operation "unpleasant".   So ... ??!

**Comment by rousskov** [comment]
> Then please clarify exactly what you meant by that last sentence.

When a core developer raises an issue about changing the copyright year, I expect the maintainer not to change the copyright year in the official branches until the issue is at least discussed. To me, it is just basic courtesy -- something that cannot be codified, but should be practiced and expected by everybody, especially folks on the core team. If others do not see how such actions can feel very unpleasant, I do not know how to explain it.

> This whole discussion gives the impression that something surprising has happened that you are only now

Unfortunately, I cannot say I was really surprised. Similar cooperation difficulties is, sadly, a recurring Squid Project theme.

**Pull request closed without merging**